### PR TITLE
Merge unnati branch to develop

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

File: elevate-self-creation-portal/controllers/orchestrationController.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const routesConfig = require('../constants/routes')
-const entityController = require('../controllers/entity')
+const entityController = require('../controllers/selfcreationportal')
```
Comment (rakeshSgr) [inline elevate-self-creation-portal/controllers/orchestrationController.js:2]: @Prajwal17Tunerlabs use the camel case for file naming convention

File: elevate-self-creation-portal/index.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ const getDependencies = () => {
 
 const getPackageMeta = () => {
 	return {
-		basePackageName: 'entity-management',
-		packageName: 'elevate-entity-management',
+		basePackageName: 'self_creation_portal',
```
Comment (rakeshSgr) [inline elevate-self-creation-portal/index.js:12]: @priyanka-TL use "-" for the package name

File: elevate-survey/controllers/survey.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+ * Fetch resources from Samiksha service.
+ * @name fetchSamikshaTemplates
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} responses  
+ * @returns {JSON} - List of resources
+ */
+/**
+ * req.body = {
+ * 		organization_id : 1,
```
Comment (rakeshSgr) [inline elevate-survey/controllers/survey.js:22]: @Prajwal17Tunerlabs please remove un wanted commented lines

File: elevate-user/package-lock.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "elevate-user",
-	"version": "1.0.12",
+	"version": "1.1.77",
```
Comment (rakeshSgr) [inline elevate-user/package-lock.json:3]: @priyanka-TL package-lock.json we can remove

File: elevate-project/controllers/project.js [hunk 5]
```diff
@@ -1,29 +1,97 @@
...
+ * Fetch project templates from projects service.
+ * @name fetchProjectTemplates
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} responses  
+ * @returns {JSON} - List of project templates
+ */
+/**
+ * req.body = {
+ * 		organization_id : 1,
```
Comment (rakeshSgr) [inline elevate-project/controllers/project.js:21]: @priyanka-TL please remove unwanted comments

File: elevate-project/controllers/project.js [hunk 6]
```diff
@@ -1,29 +1,97 @@
...
+			"projection": common.PROJECT_PROJECTION_FIELDS,
+			"limit": max_limit
+		}
+		
+		// replace the word bearer if token has it 
+		const x_auth_token = req.headers[common.AUTH_TOKEN_KEY].replace(/^(Bearer|bearer)\s*/, '');
+		// custom header 
+		const header = {
+			'internal-access-token' : req.headers['internal_access_token'],
+			'X-auth-token': x_auth_token,
```
Comment (rakeshSgr) [inline elevate-project/controllers/project.js:62]: @priyanka-TL use common.AUTH_TOKEN_KEY here

File: elevate-project/controllers/project.js [hunk 7]
```diff
@@ -1,29 +1,97 @@
...
+			'Content-Type': 'application/json'
```
Comment (rakeshSgr) [inline elevate-project/controllers/project.js:63]: @priyanka-TL as you define the content type in the common you can use the same thing here as well
HEADER_CONTENT_TYPE

File: elevate-self-creation-portal/package-lock.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "elevate-self-creation-portal",
-  "version": "1.0.0",
+  "version": "1.0.21",
```
Comment (rakeshSgr) [inline elevate-self-creation-portal/package-lock.json:3]: @Prajwal17Tunerlabs  package-locak.json not require

File: elevate-entity-management/index.js [hunk 9]
```diff
@@ -9,7 +9,7 @@ const getDependencies = () => {
 
 const getPackageMeta = () => {
 	return {
-		basePackageName: 'entity-management',
+		basePackageName: 'entity_management',
```
Comment (rakeshSgr) [inline elevate-entity-management/index.js:12]: @Prajwal17Tunerlabs keep the naming conventions  as per standards, check other packages naming convention

File: elevate-survey/controllers/survey.js (hunk 3 above)
Comment (priyanka-TL) [reply elevate-survey/controllers/survey.js:22]: @rakeshSgr Akash Asked @adithyadinesh0412  to add this

File: elevate-project/controllers/project.js (hunk 5 above)
Comment (priyanka-TL) [reply elevate-project/controllers/project.js:21]: @rakeshSgr @adithyadinesh0412  added this as per Akash comment

File: elevate-entity-management/index.js (hunk 9 above)
Comment (priyanka-TL) [reply elevate-entity-management/index.js:12]: @VISHNUDAS-tunerlabs  can u please tell, why it is changed ?

**Pull request merged**